app/assets/stylesheets/content/_tables.sass

```diff
@@ -60,14 +106,96 @@ tr
     td
       &.category
         white-space: normal
+      &.subject
+        text-align: left
+    &.idnt td.subject .icon-context:before
+      padding: 9px 0 0 10px
 
-tr.issue
-  td
-    &.subject
+  &.work-package
+    white-space: nowrap
+    &.sum
+      font-weight: bold
+    td.subject
+      white-space: normal
       text-align: left
-  &.idnt td.subject a
-    background: image-url("bullet_arrow_right.png") no-repeat 0 50%
-    padding-left: 16px
+    td.done_ratio table.progress
+      margin-left: auto
+      margin-right: auto
+    &.idnt td.subject.icon-context
+      display: inherit
+    &.issue td.category
+      white-space: normal
```

Comment:
Aren't those rules specific for the work packages table?

Shouldn't they belong into the `app/assets/stylesheets/content/_work_packages_table.sass`?

Didn't @ulferts tell me this file wouldn't be empty for long?